tests/countries/test_pitcairn_islands.py

```
@@ -0,0 +1,39 @@
+            ("2024-12-25", "Christmas Day"),
+            ("2024-12-26", "Boxing Day"),
+        )
+
+    def test_translation_pitkern(self):
+        holidays = PitcairnIslands(year=2024, language="pih")
+        self.assertEqual(holidays[date(2024, 1, 1)], "Niu Ier Dei")
+        self.assertEqual(holidays[date(2024, 1, 23)], "Bauti Dei")
+        self.assertEqual(holidays[date(2024, 12, 25)], "Krismas Dei")
+
+    def test_random_non_holiday(self):
+        self.assertNoHoliday(PitcairnIslands(year=2024), date(2024, 2, 2))
```

Ruff (0.11.9)

No newline at end of file

Add trailing newline

(W292)